tpv/core/shell.py

```diff
@@ -77,6 +101,28 @@ def create_parser():
         help="Path to the TPV config file to format. Can be a local path or http url.")
     format_parser.set_defaults(func=tpv_format_config_file)
 
+    test_parser = subparsers.add_parser(
+        'test',
+        help="Test a TPV configuration.",
+        description="")
+    test_parser.add_argument(
+        '--job-conf', type=str,
+        required=True,
+        help="Galaxy job configuration file")
+    test_parser.add_argument(
+        '--tool', type=str,
+        default='_default_',
+        help="Test mapping for Galaxy tool with given ID")
+    test_parser.add_argument(
+        '--user', type=str,
+        help="Test mapping for Galaxy user with username or email"
+    )
+    test_parser.add_argument(
+        'config',
+        nargs='+',
+        help="TPV configuration files")
```

Comment:
Since the job_conf file will include the tpv deps, we can just read them off that?

Reply:
Ah good point, I can add that. Although it doesn't really use the job conf for much, just destinations (which won't be required soon). I have been using a copy from the server that doesn't even have valid TPV config paths for my dev host. Would it be ok if we kept the command line args and use those if set, but use the job conf if not?.